ui/src/pages/domain/manage.js

```diff
@@ -115,6 +117,7 @@ export async function getServerSideProps(context) {
             manageDomains: domains[1],
             pending: domains[2],
             _csrf: domains[3],
+            test:domains[4],
```

Comment:
where are you using this test?

Reply:
I am using it for testing purpose, I using "testfive:domains[4]" to test my API.

Reply:
sorry i'm also a bit confused about this change, can you help me understand as well raja?